docs/learn/calls.md

```diff
@@ -174,6 +211,8 @@ response = recommend_book("fantasy")
 print(response.content)
 ```
 
+In this example, we're creating a custom OpenAI client with a specific API key, organization ID, and base URL. This custom client is then passed to the call decorator, allowing us to use a custom endpoint or configuration for our LLM calls.
+
 Any custom client is supported so long as it has the same API as the original base client.
 
 ## Handling Responses
```

Comment:
nit: should we link `BaseCallResponse` here to the API reference as the first mention?